evaluation/swe_bench/README.md

```diff
@@ -124,16 +126,23 @@ After running the inference, you will obtain a `output.jsonl` (by default it wil
 
 With `output.jsonl` file, you can run `eval_infer.sh` to evaluate generated patches, and produce a fine-grained report.
 
+**This evaluation is performed using the official dockerized evaluation annouced [here](https://github.com/princeton-nlp/SWE-bench/blob/main/docs/20240627_docker/README.md).**
+
 If you want to evaluate existing results, you should first run this to clone existing outputs
 
 ```bash
 git clone https://huggingface.co/spaces/OpenDevin/evaluation evaluation/evaluation_outputs
 ```
 
-To prepare for swe-bench evaluation, you should pull evaluation docker from [OpenDevin/SWE-bench-docker](https://github.com/OpenDevin/SWE-bench-docker) and download swe-bench data by running:
+If you have extra local space, you can pull the [instance-level docker images](https://github.com/princeton-nlp/SWE-bench/blob/main/docs/20240627_docker/README.md#choosing-the-right-cache_level) we've prepared to speed up the evaluation by running:
+
+```bash
+evaluation/swe_bench/scripts/docker/pull_all_eval_docker.sh instance
+```
 
+If you want to save disk space a bit while speed up the image pre-build process, you can pull the environment-level docker images:
```

Comment:
IIRC they say how much space we should expect it to take, I wonder if we can say that too, explicitly, to make it clear for readers.

Reply:
If simply calculate the sum of docker size - it will need 2T. But my disk is certainly less than 2T and it worked since there's a lot of layer re-use, which makes the number hard to estimate. Maybe we can just say 500G-1T is a safe number?

Reply:
SGTM!